packages/cordova-plugin-aerogear-metrics/plugin.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='utf-8'?>
+<plugin
+    id="cordova-plugin-aerogear-metrics"
+    version="0.0.1"
+    xmlns="http://apache.org/cordova/ns/plugins/1.0"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+>
+    <name>cordova-plugin-aerogear-metrics</name>
+
+    <js-module name="metrics" src="www/metrics.js">
+        <clobbers target="aerogear" />
```

Comment:
I think clobbers aerogear is a bad choice here since documentation says it'll be overwritten by other plugins, maybe ` <merges target="aerogear" />` or `<clobbers target="aerogear.metrics" />`

Reply:
Also as @wtrocki mentioned this could also be removed and have core use the dependant plugins directly

Reply:
Yes 2 both, I wonder if it would be useful to leave an empty `metrics.js` with the clobbers defined, just in case in the future someone needs to use it. It could be added then of course, just saying. @paolobueno @wtrocki what do you think? Leave an scaffold or kill it?